doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -57,19 +81,24 @@ The wrappers L<CMS_encrypt(3)> and L<CMS_decrypt(3)> are often used instead.
 =head1 RETURN VALUES
 
 If the allocation fails, CMS_EnvelopedData_create_ex(),
-CMS_EnvelopedData_create(), CMS_AuthEnvelopedData_create_ex(), and
-CMS_AuthEnvelopedData_create() return NULL and set an error code that can be
-obtained by L<ERR_get_error(3)>. Otherwise they return a pointer to the newly
+CMS_EnvelopedData_create(), CMS_AuthEnvelopedData_create_ex()
+CMS_AuthEnvelopedData_create(), CMS_AuthEnvelopedData_create(),
+CMS_AuthEnvelopedData_create_ex() and CMS_sign_encrypt() return NULL and set an error code
+that can be obtained by L<ERR_get_error(3)>. Otherwise they return a pointer to the newly
 allocated structure.
```

Comment:
```suggestion
CMS_AuthEnvelopedData_create_ex(), and CMS_sign_encrypt() 
return NULL and set an error code that can be obtained by L<ERR_get_error(3)>. 
Otherwise, they return a pointer to the newly allocated structure.
```